miden-lib/build.rs

```diff
@@ -17,12 +66,44 @@ fn main() -> io::Result<()> {
     // or its builder changed:
     println!("cargo:rerun-if-changed=asm");
```

Comment:
Not due to of this PR, but line 65 has a typo in the comment (i.e., incorrect `.masl` file is mentioned).

Reply:
re-worded the comment